src/modules/fix_widescreen.rs

```diff
@@ -0,0 +1,70 @@
+//! `bxt_fix_widescreen_fov`
+
+use super::Module;
+use crate::hooks::engine;
+use crate::modules::cvars::CVar;
+use crate::utils::*;
+
+pub struct FixWidescreen;
+impl Module for FixWidescreen {
+    fn name(&self) -> &'static str {
+        "bxt_fix_widescreen_fov"
+    }
+
+    fn description(&self) -> &'static str {
+        "Fixes widescreen vertical field-of-view."
```

Comment:
```suggestion
        "Correcting widescreen vertical field-of-view."
```

Reply:
yea I think this makes sense because it does not really fix anything as it also changes horizontal FOV. For a while I thought that is how it works because it just looks wide.